### PR TITLE
Fix typo in "Optimize the PDF" preamble

### PR DESCRIPTION
**Pull request by mcp292**

### CODE REVIEW

File: docs/modules/ROOT/pages/optimize-pdf.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 = Optimize the PDF
 :url-hexapdf: https://hexapdf.gettalong.org/
 
-By default, Asciidoctor PDF does not optimize the PDF it generates or compresses its streams.
+By default, Asciidoctor PDF does not optimize the PDF it generates, nor compress its streams.
```
Comment (mcp292) [inline docs/modules/ROOT/pages/optimize-pdf.adoc:4]:
```suggestion
By default, Asciidoctor PDF does not optimize nor compress the PDF it generates.
```

**Review by mcp292 — COMMENTED** [review]
It could also be simplified to this. Wasn't sure if this loses information.

File: docs/modules/ROOT/pages/optimize-pdf.adoc (hunk 1 above)
Comment (mojavelinux) [reply docs/modules/ROOT/pages/optimize-pdf.adoc:4]: Technically, the original sentence is correct. Adding nor is a double negative and thus incorrect grammar. However, this is a case where the correct grammar is hard to read. Let's go with the verbose form.

> By default, Asciidoctor PDF does not optimize the PDF it generates and it does not compress its streams.

Technically, optimization is different than compressing streams, which is why they are listed separately.

**Comment by mojavelinux** [comment]
Applied as 65235f20cf03f054a4c7e94c433c22d13f731027

File: docs/modules/ROOT/pages/optimize-pdf.adoc (hunk 1 above)
Comment (mcp292) [reply docs/modules/ROOT/pages/optimize-pdf.adoc:4]: I should've been more clear about what the "typo" was. I was referring to "compresses" in:

> does not optimize the PDF... or **compresses** its streams

So maybe the original could have been fixed by changing "compresses" to "compress":
```diff
- By default, Asciidoctor PDF does not optimize the PDF it generates or compresses its streams.
+ By default, Asciidoctor PDF does not optimize the PDF it generates or compress its streams.
```

That said, verbose is explicit and explicit is good. I have no objections. Thanks for applying.

Comment (mojavelinux) [reply docs/modules/ROOT/pages/optimize-pdf.adoc:4]: I'll admit I didn't notice the typo fix at first. But when I was looking at the diff when committing it, I finally spotted it. I can't believe that went unnoticed for so long. Typos are sure good at hiding in plain sight.

Comment (mcp292) [reply docs/modules/ROOT/pages/optimize-pdf.adoc:4]: Sure are! Glad to help!

**Pull request closed without merging**